pkg/metrics/metrics.go

```diff
@@ -474,6 +482,26 @@ func UpdateKafkaRequestsStatusSinceCreatedMetric(status constants2.KafkaStatus,
 	kafkaStatusSinceCreatedMetric.With(labels).Set(elapsed.Seconds())
 }
 
+// create a new GaugeVec for kafkas status duration
+var kafkaStatusSinceLastSetMetric = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Subsystem: KasFleetManager,
+		Name:      KafkaRequestsStatusSinceLastSet,
+		Help:      "metrics to track the status of a Kafka instance and the timestamp of the last update",
```

Comment:
@pawelpaszki how do you see the query for getting the duration of how long a Kafka is in its current state using this metric?

For getting the current status, the only way I can think of is to get the metric with the value of the latest timestamp, maybe there's a way we can do that with promql :thinking:. Otherwise, is there some other metric we can use together with this one to do that?

Reply:
I was under the impression that we could compare timestamp to current time, but not sure if its even possible